src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/RemoteTagHelperProviderServiceFactory.cs

```diff
@@ -13,5 +14,5 @@ public RemoteTagHelperProviderServiceFactory() : base(RazorServiceDescriptors.Ta
     }
 
     protected override IRemoteTagHelperProviderService CreateService(IServiceBroker serviceBroker)
-            => new RemoteTagHelperProviderService(serviceBroker);
```

Comment:
This is the main fix. Instead of relying on MEF composition we just create a new reporter for the OOP process.